resources/views/includes/header_employee.blade.php

```diff
@@ -0,0 +1,36 @@
+<nav class="navbar navbar-default">
```

Comment:
Fix the routes here as well - use named routes with the `route()` function instead of static urls, or the `url` function which also works with static urls.